README.md

```diff
@@ -1 +1,6 @@
-# Hello_Microverse
\ No newline at end of file
+ Hello_Microverse 
+
+ Description 
+
+ This is Microverse's Student First Hello Microverse repo assignment Includes The Dependency Installment aimed to Familiarize Each Student With Various Packages Set up.
+
```

Comment:
### Great work implementing this project. I know you put in a lot of work to get it through. Kudos 👏 💪

- [ ] You submitted this project without a well-documented README file. Microverse has a [new README template](https://github.com/microverseinc/readme-template/blob/master/README.md) that all students should use and edit to suit the particular project. Kindly use this template to create a professional README.md file https://github.com/microverseinc/readme-template. For this template attached, kindly do the following when customizing it to suit this project:

- Remove the Microverse logo. If you have your logo, you can use it but I would advise you to remove it. 🙏
- For the Tech Stack Section, Kindly remove the Server and Database subsections because the technologies listed there are not relevant to this project. For the Client section, kindly include HTML and CSS 👍
- For the Key Features section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file. 👍
- For the Prerequisite Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:

    - Installation of Git
    - Installation of NPM
    - A working browser
    - Internet connection
    - A code editor like Visual Studio Code
    - Installation of Live Server plugging in Visual Studio Code
    - Any other tool you can think about

- For the Setup section, kindly add instructions for cloning your repository on a local machine 👍
- For the Install section, kindly add instructions for setting up linters used in this project
- For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.
- You can leave out the test and deployment section since they are not relevant to this project.  
- For the MIT section, kindly add a valid LICENSE. You can follow the instructions on the right side of the article attached below to set up a valid LICENSE
https://choosealicense.com/licenses/mit/
- The future features section is also necessary, kindly update it with at least three items.

Kindly implement all the changes listed above. It will make your README.md file look professional and it will give you an idea of how to set README.md files for most of the projects you will execute as a Microverse student. 👍 💻

Reply:
Thanks a bunch for the suggestions